polynote-frontend/polynote/cell.js

```diff
@@ -395,7 +396,7 @@ export class CodeCell extends Cell {
             if (result.name !== "Out") {
                 // don't display this; it's a named declaration
                 // TODO: have a way to display these if desired
-            } else {
+            } else if (!this.resultEl || !this.resultEl.reprs || !this.resultEl.reprs.length || this.resultEl.reprs.length < result.reprs.length) {
```

Comment:
@jeremyrsmith this is the only change to `Out` that I could find, could something here have caused the missing `Out`s?

Reply:
Yeah, not sure how I didn't encounter it before... but there is a case when `this.resultEl` is not falsy but also is not in the DOM tree. Will fix with an extra check I suppose.

The point of this code is to be tolerant of the server sending multiple `Out` results for a cell, i.e. if there's one in the notebook itself and one in the symbol table. Maybe that's the place to solve the problem instead.